extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js

```diff
@@ -15,6 +15,7 @@ export default class StaticWadoClient extends api.DICOMwebClient {
     "StudyDescription": "00081030",
     "StudyDate": "00080020",
     "ModalitiesInStudy": "00080061",
+    AccessionNumber: "00080050",
```

Comment:
Need to list all the query fields in order to pass the e2e tests.  This fixes 1 test.